motion-led-controller/Makefile

```diff
@@ -38,14 +38,14 @@ define Build/Clean
 	$(call Build/Compile/Default,clean)
 endef
 
-define Package/light-controller/install
+define Package/motion-led-controller/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/set_led.sh $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/light_controller_continuous.sh $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/motion_led_controller_continuous.sh $(1)/usr/bin
 	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/light_controller_appd $(1)/etc/init.d/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/light_controller_continuousd $(1)/etc/init.d/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/motion_led_controller_appd $(1)/etc/init.d/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/motion_led_controller_continuousd $(1)/etc/init.d/
 endef
```

Comment:
file names in motion-led-controller github repo also need to be changed to help verify this PR